src/ValueFormatters/QuantityFormatter.php

```diff
@@ -63,14 +63,26 @@ class QuantityFormatter extends ValueFormatterBase {
 	private $vocabularyUriFormatter;
 
 	/**
+	 * @var string
+	 */
+	private $quantityWithUnitFormat;
+
+	/**
+	 * @since 0.6
+	 *
+	 * @param FormatterOptions|null $options
 	 * @param DecimalFormatter|null $decimalFormatter
 	 * @param ValueFormatter|null $vocabularyUriFormatter
-	 * @param FormatterOptions|null $options
+	 * @param string|null $quantityWithUnitFormat Format string with two placeholders, $1 for the
+	 * number and $2 for the unit. Warning, this must be under the control of the application, not
+	 * under the control of the user, because it allows HTML injections in subclasses that return
+	 * HTML.
 	 */
 	public function __construct(
+		FormatterOptions $options = null,
```

Comment:
I have split this change off to #45. Please merge #45 first and let me know, I will rebase #44 then.